engine/src/main/java/org/terasology/rendering/nui/widgets/UIList.java

```diff
@@ -53,6 +54,9 @@
     private Binding<List<T>> list = new DefaultBinding<>(new ArrayList<>());
     private ItemRenderer<T> itemRenderer = new ToStringTextRenderer<>();
     private Binding<Boolean> canBeFocus = new DefaultBinding<>(true);
+    private int listMin = -1;
```

Comment:
Why is this initialised to `-1` if it's set to `0` on the first frame via `onDraw`